travis-run-tests.sh

```diff
@@ -25,5 +25,10 @@ if [ $1 == "group4" ]; then
   cd ../app-springboot
   mvn -B -s ../maven-settings.xml clean test
 fi
```

Comment:
@sebastienblanc out of curiosity. Why you have 2 ifs for `group5`?

Reply:
`productize.sh` should not run for Pull Requests.

Reply:
@sebastienblanc why not something like:
```
if [ $1 == "group5" ] && [ $TRAVIS_PULL_REQUEST == "false" ]; then
 ./productize.sh
  exit 0
fi  
```

???

Reply:
yeah, probably to my incremental experiences, the unecessary condition has not been removed, let me try your suggestion.

Reply:
Yeah let me try that, the extra condition was probably there after my (many) expirements.